.eslintrc.json

```diff
@@ -0,0 +1,136 @@
+{
+  "root": true,
+
+  "env": {
+    "node": true
+  },
+
+  "globals": {},
+
+  "rules": {
+    // Rules are divided into sections from http://eslint.org/docs/rules/
+
+    // Possible errors
+    "comma-dangle": ["error", "never"],
+    "no-cond-assign": ["error", "except-parens"],
+    "no-constant-condition": "error",
+    "no-control-regex": "error",
+    "no-debugger": "error",
+    "no-dupe-args": "error",
+    "no-dupe-keys": "error",
+    "no-duplicate-case": "error",
+    "no-empty-character-class": "error",
+    "no-empty": "error",
+    "no-ex-assign": "error",
+    "no-extra-boolean-cast": "error",
+    "no-extra-semi": "error",
+    "no-func-assign": "error",
+    "no-inner-declarations": "error",
+    "no-invalid-regexp": "error",
+    "no-irregular-whitespace": "error",
+    "no-negated-in-lhs": "error",
+    "no-obj-calls": "error",
+    "no-regex-spaces": "error",
+    "no-sparse-arrays": "error",
+    "no-unreachable": "error",
+    "use-isnan": "error",
+    "no-unsafe-finally": "error",
+    "valid-typeof": "error",
+    "no-unexpected-multiline": "error",
+
+    // Best practices
+    "accessor-pairs": "error",
+    "array-callback-return": "error",
+    "complexity": ["warn", 10],
+//    "dot-notation": "error", // TODO enable?
```

Comment:
I think there are places where we deliberately use bracket-notation. If there are many such places (that need a "manual" override), I would leave this disabled.